TROUBLESHOOTING.md

```diff
@@ -0,0 +1,42 @@
+# Troubleshooting
+
+You may find that the Terraform Provider doesn't always provide the most useful error messages.
+Here are some common errors and their workarounds.
+
+## unexpected state 'CART'
+
+```text
+$ terraform destroy
+... snip ...
+cloudbolt_bp_instance.mycbresource: Destroying... [id=/api/v2/resources/service/35/]
+cloudbolt_bp_instance.mycbresource: Still destroying... [id=/api/v2/resources/service/35/, 10s elapsed]
+
+Error: Error waiting for Job (/api/v2/orders/81/) to complete: unexpected state 'CART', wanted target 'SUCCESS'. last error: %!s(<nil>)
+```
+
+This means that the "Delete" resource action requires approval in CloudBolt.
+
+To resolve this...
+
+1. Go to `https://your.cloudbolt.instance/actions/resource_actions/`
```

Comment:
I have seen people read URLs like this literally. Would the following make this more obvious?

```
1. Go to `https://<your-cloudbolt-instance>/actions/resource_actions/`
```

Reply:
```suggestion
1. Go to `https://<your-cloudbolt-instance>/actions/resource_actions/`
```